homeassistant/components/fronius/sensor.py

```diff
@@ -77,225 +79,240 @@ async def async_setup_entry(
         )
 
 
-INVERTER_ENTITY_DESCRIPTIONS: list[SensorEntityDescription] = [
-    SensorEntityDescription(
+@dataclass
+class FroniusSensorEntityDescription(SensorEntityDescription):
+    """Describes Fronius sensor entity."""
+
+    default_value: StateType | None = None
```

Comment:
Can this logic by moved into the library instead?

Reply:
The lib currently does only parsing of the response without applying logic. 

Right now we make use of the fact that some keys are not included in a response by not creating an entity as long as it's key wasn't seen. Eg. in an installation without battery, you wouldn't get battery related entities. If we used default values in the lib this wouldn't work anymore and we would probably create stale entities (without treating these special cases in the integration again).